### ISSUE TITLE
PAPER type doesn't start

### ISSUE THREAD

**Issue by LapinFou** [itzg/docker-minecraft-server#2448]
### Describe the problem

With the latest version (downloaded today 2023-10-21), the docker doesn't start anymore. It is locked in an init loop.
I removed completely the container and recreated it.

### Container definition

Using Portainer with `stack.env`.
```
version: "3.8"
services:
  minecraft:
    image: itzg/minecraft-server:latest
    hostname: minecraftjava
    container_name: minecraftjava
    env_file:
      - stack.env
    volumes:
      - /volume1/docker/minecraftjava:/data:rw
    ports:
      - 25565:25565/udp
      - 25565:25565/tcp
    restart: unless-stopped

```
`stack.env` variables list:
```
UID=1055
GID=100
TYPE=PAPER
SERVER_NAME=LaFamilia
JVM_XX_OPTS=-XX:+UseG1GC -XX:+ParallelRefProcEnabled -XX:MaxGCPauseMillis=200 -XX:+UnlockExperimentalVMOptions -XX:+DisableExplicitGC -XX:+AlwaysPreTouch -XX:G1HeapWastePercent=5 -XX:G1MixedGCCountTarget=4 -XX:G1MixedGCLiveThresholdPercent=90 -XX:G1RSetUpdatingPauseTimePercent=5 -XX:SurvivorRatio=32 -XX:+PerfDisableSharedMem -XX:MaxTenuringThreshold=1 -XX:G1NewSizePercent=30 -XX:G1MaxNewSizePercent=40 -XX:G1HeapRegionSize=8M -XX:G1ReservePercent=20 -XX:InitiatingHeapOccupancyPercent=15
MAX_MEMORY=4G
EULA=true
MOTD="Serveur Minecraft §c§lLa Familia§r"
TZ=Europe/Paris
PVP=false
```

### Container logs

```
[init] Changing uid of minecraft to 1055
[init] Changing gid of minecraft to 100
[init] Running as uid=1055 gid=100 with /data as 'drwxrwxrwx 1 1055 100 662 Oct 15 19:41 /data'
[init] Resolving type given PAPER
[mc-image-helper] 18:17:40.015 WARN  : [40884afd-1, L:/172.24.0.2:56201 - R:api.papermc.io/172.67.72.198:443] The connection observed an error
io.netty.handler.timeout.ReadTimeoutException: null
[mc-image-helper] 18:17:40.032 ERROR : 'install-paper' command failed. Version is 1.36.4
io.netty.handler.timeout.ReadTimeoutException: null
[init] ERROR: failed to download paper
[init] Running as uid=1055 gid=100 with /data as 'drwxrwxrwx 1 1055 100 662 Oct 15 19:41 /data'
[init] Resolving type given PAPER
[mc-image-helper] 18:18:20.744 WARN  : [c10dbcd8-1, L:/172.24.0.2:54852 - R:api.papermc.io/104.26.13.138:443] The connection observed an error
io.netty.handler.timeout.ReadTimeoutException: null
[mc-image-helper] 18:18:20.766 ERROR : 'install-paper' command failed. Version is 1.36.4
io.netty.handler.timeout.ReadTimeoutException: null
[init] ERROR: failed to download paper
[init] Running as uid=1055 gid=100 with /data as 'drwxrwxrwx 1 1055 100 662 Oct 15 19:41 /data'
[init] Resolving type given PAPER
[mc-image-helper] 18:19:01.882 WARN  : [2ac13cf5-1, L:/172.24.0.2:56227 - R:api.papermc.io/172.67.72.198:443] The connection observed an error
io.netty.handler.timeout.ReadTimeoutException: null
[mc-image-helper] 18:19:01.902 ERROR : 'install-paper' command failed. Version is 1.36.4
io.netty.handler.timeout.ReadTimeoutException: null
[init] ERROR: failed to download paper
[init] Running as uid=1055 gid=100 with /data as 'drwxrwxrwx 1 1055 100 662 Oct 15 19:41 /data'
[init] Resolving type given PAPER
[mc-image-helper] 18:19:12.533 ERROR : 'install-paper' command failed. Version is 1.36.4
me.itzg.helpers.http.FailedRequestException: HTTP request of https://api.papermc.io/v2/projects/paper failed with 520 : Fetching object content
	at me.itzg.helpers.http.FetchBuilderBase.lambda$failedRequestMono$2(FetchBuilderBase.java:195)
	at reactor.core.publisher.MonoFlatMap$FlatMapMain.onNext(MonoFlatMap.java:132)
	at reactor.core.publisher.Operators$BaseFluxToMonoOperator.completePossiblyEmpty(Operators.java:2071)
	at reactor.core.publisher.FluxDefaultIfEmpty$DefaultIfEmptySubscriber.onComplete(FluxDefaultIfEmpty.java:134)
	at reactor.core.publisher.FluxHandle$HandleSubscriber.onComplete(FluxHandle.java:223)
	at reactor.core.publisher.FluxMap$MapConditionalSubscriber.onComplete(FluxMap.java:275)
	at reactor.core.publisher.FluxDoFinally$DoFinallySubscriber.onComplete(FluxDoFinally.java:128)
	at reactor.core.publisher.FluxHandleFuseable$HandleFuseableSubscriber.onNext(FluxHandleFuseable.java:211)
	at reactor.core.publisher.FluxContextWrite$ContextWriteSubscriber.onNext(FluxContextWrite.java:107)
	at reactor.core.publisher.Operators$BaseFluxToMonoOperator.completePossiblyEmpty(Operators.java:2071)
	at reactor.core.publisher.MonoCollectList$MonoCollectListSubscriber.onComplete(MonoCollectList.java:118)
	at reactor.core.publisher.FluxPeek$PeekSubscriber.onComplete(FluxPeek.java:260)
	at reactor.core.publisher.FluxMap$MapSubscriber.onComplete(FluxMap.java:144)
	at reactor.netty.channel.FluxReceive.onInboundComplete(FluxReceive.java:413)
	at reactor.netty.channel.ChannelOperations.onInboundComplete(ChannelOperations.java:444)
	at reactor.netty.channel.ChannelOperations.terminate(ChannelOperations.java:498)
	at reactor.netty.http.client.HttpClientOperations.onInboundNext(HttpClientOperations.java:753)
	at reactor.netty.channel.ChannelOperationsHandler.channelRead(ChannelOperationsHandler.java:114)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:444)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412)
	at io.netty.handler.timeout.IdleStateHandler.channelRead(IdleStateHandler.java:286)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:442)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412)
	at io.netty.channel.CombinedChannelDuplexHandler$DelegatingChannelHandlerContext.fireChannelRead(CombinedChannelDuplexHandler.java:436)
	at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:346)
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:318)
	at io.netty.channel.CombinedChannelDuplexHandler.channelRead(CombinedChannelDuplexHandler.java:251)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:442)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412)
	at io.netty.handler.ssl.SslHandler.unwrap(SslHandler.java:1471)
	at io.netty.handler.ssl.SslHandler.decodeJdkCompatible(SslHandler.java:1334)
	at io.netty.handler.ssl.SslHandler.decode(SslHandler.java:1383)
	at io.netty.handler.codec.ByteToMessageDecoder.decodeRemovalReentryProtection(ByteToMessageDecoder.java:529)
	at io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:468)
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:290)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:444)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412)
	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1410)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:440)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)
	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:919)
	at io.netty.channel.epoll.AbstractEpollStreamChannel$EpollStreamUnsafe.epollInReady(AbstractEpollStreamChannel.java:800)
	at io.netty.channel.epoll.EpollEventLoop.processReady(EpollEventLoop.java:509)
	at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:407)
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Unknown Source)
	Suppressed: java.lang.Exception: #block terminated with an error
		at reactor.core.publisher.BlockingSingleSubscriber.blockingGet(BlockingSingleSubscriber.java:103)
		at reactor.core.publisher.Mono.block(Mono.java:1712)
		at me.itzg.helpers.paper.InstallPaperCommand.useCoordinates(InstallPaperCommand.java:152)
		at me.itzg.helpers.paper.InstallPaperCommand.call(InstallPaperCommand.java:109)
		at me.itzg.helpers.paper.InstallPaperCommand.call(InstallPaperCommand.java:35)
		at picocli.CommandLine.executeUserObject(CommandLine.java:2041)
		at picocli.CommandLine.access$1500(CommandLine.java:148)
		at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2461)
		at picocli.CommandLine$RunLast.handle(CommandLine.java:2453)
		at picocli.CommandLine$RunLast.handle(CommandLine.java:2415)
		at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2273)
		at picocli.CommandLine$RunLast.execute(CommandLine.java:2417)
		at picocli.CommandLine.execute(CommandLine.java:2170)
		at me.itzg.helpers.McImageHelper.main(McImageHelper.java:157)
[init] ERROR: failed to download paper
[init] Running as uid=1055 gid=100 with /data as 'drwxrwxrwx 1 1055 100 662 Oct 15 19:41 /data'
[init] Resolving type given PAPER
[mc-image-helper] 18:19:53.058 WARN  : [0db93015-1, L:/172.24.0.2:54884 - R:api.papermc.io/104.26.13.138:443] The connection observed an error
io.netty.handler.timeout.ReadTimeoutException: null
[mc-image-helper] 18:19:53.077 ERROR : 'install-paper' command failed. Version is 1.36.4
io.netty.handler.timeout.ReadTimeoutException: null
[init] ERROR: failed to download paper
[init] Running as uid=1055 gid=100 with /data as 'drwxrwxrwx 1 1055 100 662 Oct 15 19:41 /data'
[init] Resolving type given PAPER
[mc-image-helper] 18:20:33.293 WARN  : [3d139cc3-1, L:/172.24.0.2:56257 - R:api.papermc.io/172.67.72.198:443] The connection observed an error
io.netty.handler.timeout.ReadTimeoutException: null
[mc-image-helper] 18:20:33.313 ERROR : 'install-paper' command failed. Version is 1.36.4
io.netty.handler.timeout.ReadTimeoutException: null
[init] ERROR: failed to download paper
[init] Running as uid=1055 gid=100 with /data as 'drwxrwxrwx 1 1055 100 662 Oct 15 19:41 /data'
[init] Resolving type given PAPER
[mc-image-helper] 18:21:14.177 WARN  : [f973749c-1, L:/172.24.0.2:42923 - R:api.papermc.io/104.26.12.138:443] The connection observed an error
io.netty.handler.timeout.ReadTimeoutException: null
[mc-image-helper] 18:21:14.196 ERROR : 'install-paper' command failed. Version is 1.36.4
io.netty.handler.timeout.ReadTimeoutException: null
[init] ERROR: failed to download paper
[init] Running as uid=1055 gid=100 with /data as 'drwxrwxrwx 1 1055 100 662 Oct 15 19:41 /data'
[init] Resolving type given PAPER
[mc-image-helper] 18:21:54.345 WARN  : [f37f7f57-1, L:/172.24.0.2:54924 - R:api.papermc.io/104.26.13.138:443] The connection observed an error
io.netty.handler.timeout.ReadTimeoutException: null
[mc-image-helper] 18:21:54.366 ERROR : 'install-paper' command failed. Version is 1.36.4
io.netty.handler.timeout.ReadTimeoutException: null
[init] ERROR: failed to download paper
[init] Running as uid=1055 gid=100 with /data as 'drwxrwxrwx 1 1055 100 662 Oct 15 19:41 /data'
[init] Resolving type given PAPER

```

**Comment by XgamerTV** [comment]
Same for me also just stopped working

**Comment by Verggy** [comment]
mine too, seems like same issue, any ideas?

**Comment by Verggy** [comment]
papermc server is down, how can i force my docker container to actually use already existing server file instead of failing to download new one from papermc?

**Comment by XgamerTV** [comment]
Dont know but we can close this as PaperMC is working on it

**Comment by LapinFou** [comment]
Indeed the problem is gone.
```
[init] Running as uid=1055 gid=100 with /data as 'drwxrwxrwx 1 1055 100 662 Oct 15 19:41 /data'
[init] Resolving type given PAPER
[mc-image-helper] 01:25:37.305 INFO  : Resolved paper to version 1.20.2 build 243
[mc-image-helper] 01:27:28.489 INFO  : Downloaded /data/paper-1.20.2-243.jar
[mc-image-helper] 01:27:28.636 INFO  : Removing old file paper-1.20.2-240.jar
[mc-image-helper] 01:27:52.299 INFO  : Created/updated 1 property in /data/server.properties
[init] Setting initial memory to 1G and max to 4G
[init] Starting the Minecraft server...
Applying patches
Starting org.bukkit.craftbukkit.Main
System Info: Java 17 (OpenJDK 64-Bit Server VM 17.0.8.1+1) Host: Linux 3.10.108 (amd64)
Loading libraries, please wait...
2023-10-22 01:29:03,874 ServerMain WARN Advanced terminal features are not available in this environment
[01:29:34 INFO]: Environment: Environment[accountsHost=https://api.mojang.com, sessionHost=https://sessionserver.mojang.com, servicesHost=https://api.minecraftservices.com, name=PROD]
[01:29:48 INFO]: Loaded 7 recipes
[01:29:51 INFO]: Starting minecraft server version 1.20.2
[01:29:51 INFO]: Loading properties
[01:29:52 INFO]: This server is running Paper version git-Paper-243 (MC: 1.20.2) (Implementing API version 1.20.2-R0.1-SNAPSHOT) (Git: 3a5c6f8)
```
